shinano-l.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" ?>
+<manifest>
+
+  <include name="base-l-aosp.xml"/>
+
+  <default remote="caf" revision="refs/tags/android-5.1.1_r1" sync-j="4"/>
+
+  <!-- Platform common things -->
+  <project name="device/generic/armv7-a-neon" path="device/generic/armv7-a-neon"/>
+  <project name="hardware_qcom_display" path="hardware/qcom/display" remote="b2g" revision="b2g-5.1.0_r1"/>
+  <project name="init_sh" path="external/init_sh" remote="b2g" revision="master"/>
```

Comment:
It would help readability if repos used as is from upstream were separated from those we fork.